# HG changeset patch
# User Vladimir Chalupecky <vladimir.chalupecky@gmail.com>
# Date 1402537862 -32400
# Branch vladimir_ch/bug-796-fix-eigen3config.cmake
# Node ID 21fbeccc596f3c998747802a15aa3ebb9e840f27
# Parent  8b9458389e0ce57a4aaf88e6690626ea7cf4e217
Change variable names in Eigen3Config.cmake to EIGEN3_*

diff --git a/cmake/Eigen3Config.cmake.in b/cmake/Eigen3Config.cmake.in
--- a/cmake/Eigen3Config.cmake.in
+++ b/cmake/Eigen3Config.cmake.in
@@ -3,26 +3,26 @@
 #  Eigen3Config.cmake(.in)
 
 # Use the following variables to compile and link against Eigen:
-#  EIGEN_FOUND              - True if Eigen was found on your system
-#  EIGEN_USE_FILE           - The file making Eigen usable
-#  EIGEN_DEFINITIONS        - Definitions needed to build with Eigen
-#  EIGEN_INCLUDE_DIR        - Directory where signature_of_eigen3_matrix_library can be found
-#  EIGEN_INCLUDE_DIRS       - List of directories of Eigen and it's dependencies
-#  EIGEN_ROOT_DIR           - The base directory of Eigen
-#  EIGEN_VERSION_STRING     - A human-readable string containing the version
-#  EIGEN_VERSION_MAJOR      - The major version of Eigen
-#  EIGEN_VERSION_MINOR      - The minor version of Eigen
-#  EIGEN_VERSION_PATCH      - The patch version of Eigen
+#  EIGEN3_FOUND              - True if Eigen was found on your system
+#  EIGEN3_USE_FILE           - The file making Eigen usable
+#  EIGEN3_DEFINITIONS        - Definitions needed to build with Eigen
+#  EIGEN3_INCLUDE_DIR        - Directory where signature_of_eigen3_matrix_library can be found
+#  EIGEN3_INCLUDE_DIRS       - List of directories of Eigen and it's dependencies
+#  EIGEN3_ROOT_DIR           - The base directory of Eigen
+#  EIGEN3_VERSION_STRING     - A human-readable string containing the version
+#  EIGEN3_VERSION_MAJOR      - The major version of Eigen
+#  EIGEN3_VERSION_MINOR      - The minor version of Eigen
+#  EIGEN3_VERSION_PATCH      - The patch version of Eigen
 
-set ( EIGEN_FOUND 1 )
-set ( EIGEN_USE_FILE     "@EIGEN_USE_FILE@" )
+set ( EIGEN3_FOUND 1 )
+set ( EIGEN3_USE_FILE     "@EIGEN_USE_FILE@" )
 
-set ( EIGEN_DEFINITIONS  "@EIGEN_DEFINITIONS@" )
-set ( EIGEN_INCLUDE_DIR  "@EIGEN_INCLUDE_DIR@" )
-set ( EIGEN_INCLUDE_DIRS "@EIGEN_INCLUDE_DIRS@" )
-set ( EIGEN_ROOT_DIR     "@EIGEN_ROOT_DIR@" )
+set ( EIGEN3_DEFINITIONS  "@EIGEN_DEFINITIONS@" )
+set ( EIGEN3_INCLUDE_DIR  "@EIGEN_INCLUDE_DIR@" )
+set ( EIGEN3_INCLUDE_DIRS "@EIGEN_INCLUDE_DIRS@" )
+set ( EIGEN3_ROOT_DIR     "@EIGEN_ROOT_DIR@" )
 
-set ( EIGEN_VERSION_STRING "@EIGEN_VERSION_STRING@" )
-set ( EIGEN_VERSION_MAJOR  "@EIGEN_VERSION_MAJOR@" )
-set ( EIGEN_VERSION_MINOR  "@EIGEN_VERSION_MINOR@" )
-set ( EIGEN_VERSION_PATCH  "@EIGEN_VERSION_PATCH@" )
+set ( EIGEN3_VERSION_STRING "@EIGEN_VERSION_STRING@" )
+set ( EIGEN3_VERSION_MAJOR  "@EIGEN_VERSION_MAJOR@" )
+set ( EIGEN3_VERSION_MINOR  "@EIGEN_VERSION_MINOR@" )
+set ( EIGEN3_VERSION_PATCH  "@EIGEN_VERSION_PATCH@" )
